src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px 40px;
+}
+
+.card__elem {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
+}
+
+.content__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.content__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 0 0 16px;
+  line-height: 14px;
+}
+
+.content-rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 4px;
+  height: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  margin-bottom: 24px;
+}
+
+.content-rating__stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(images/star-active.svg) no-repeat center;
+
+}
+
+.stars-star--5 {
+  background: url(images/star.svg) no-repeat center;
+
+}
```

Comment:
no, it shouldn't be like that, this modifier `stars--n` should be applied to `stars` block. And you should specify all of `n` values